### PR TITLE
Update 2023-01-25-htcondor-osg.md

### PR DESCRIPTION
**Pull request by jstathas**

### CODE REVIEW

File: _events/2023-01-25-htcondor-osg.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-short_title: Joint OSG All Hands Meeting and HTCondor Week July 10-14
-title: "Save the Dates for the Joint HTCondor Week - OSG All-Hands Meeting - 10-14 July 2023"
+short_title: Announcing Throughput Computing 2023 to be held July 10-14
+title: "Save the Dates for Throughput Computing 2023 - a joint HTCondor OSG All-Hands event"
 layout: events
 published: true
 
 excerpt: |
-    Joint OSG All Hands Meeting and HTCondor Week July 10-14 2023
+    Save the Dates for **Throughput Computing 2023** - a joint OSG All Hands Meeting and HTCondor Week to be held July 10-14 2023 in Madison, Wisconsin
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:8]: Janet, it should always be “OSG All-Hands Meeting” — exactly like that, with the hyphen. This is our brand name, so just like any brand name, we want it to appear the same way every time.

File: _events/2023-01-25-htcondor-osg.md [hunk 2]
```diff
@@ -21,13 +21,14 @@ header_image: "/images/capital.jpeg"
 <p style="font-size: larger; font-weight: bold;">Don't miss these in-person learning opportunities in beautiful Madison, Wisconsin!</p>
 
 
-Save the dates! For the first time, the OSG Consortium, the HTCondor team, and the Center for High Throughput Computing will be hosting a Joint HTCondor Week - OSG All-Hands Meeting from July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/).
+Save the dates for **Throughput Computing 2023!** For the first time, we will be offerring **Throughput Computing 2023**, which combines the much loved events **HTCondor Week** and the **OSG All-Hands Meeting** into a new joint event to be held July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/). **Througput Computing 2023** is sponsored by the OSG Consortium, the HTCondor team, and the UW-Madison Center for High Throughput Computing.
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:24]: “Throughput” is misspelled the last time.

File: _events/2023-01-25-htcondor-osg.md [hunk 3]
```diff
@@ -21,13 +21,14 @@ header_image: "/images/capital.jpeg"
...
 
 
-Save the dates! For the first time, the OSG Consortium, the HTCondor team, and the Center for High Throughput Computing will be hosting a Joint HTCondor Week - OSG All-Hands Meeting from July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/).
+Save the dates for **Throughput Computing 2023!** For the first time, we will be offerring **Throughput Computing 2023**, which combines the much loved events **HTCondor Week** and the **OSG All-Hands Meeting** into a new joint event to be held July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/). **Througput Computing 2023** is sponsored by the OSG Consortium, the HTCondor team, and the UW-Madison Center for High Throughput Computing. 
 
-Our hope is that this will primarily be an in-person event, but remote participation (via Zoom) for plenary events will also be offered. Required registration for both components will open in March 2023.
+Our hope is that this will primarily be an in-person event, but remote participation (via Zoom) for the many plenary events will also be offered. Required registration for both components will open in March 2023.
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:26]: “Our hope”? I thought it was long-ago decided that this *is* primarily an in-person event.

File: _events/2023-01-25-htcondor-osg.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
 ---
-short_title: Joint OSG All Hands Meeting and HTCondor Week July 10-14
-title: "Save the Dates for the Joint HTCondor Week - OSG All-Hands Meeting - 10-14 July 2023"
+short_title: Announcing Throughput Computing 2023 to be held July 10-14
+title: "Save the Dates for Throughput Computing 2023 - a joint HTCondor OSG All-Hands event"
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:3]: “a joint HTCondor OSG All-Hands event” doesn’t make sense to me. I would say it is either “a joint HTCondor/OSG event” or spell out the whole thing as “a joint HTCondor Week / OSG All-Hands Meeting event”. Given that we say the latter enough and this is a title, I would vote for the shorter version.

File: _events/2023-01-25-htcondor-osg.md [hunk 5]
```diff
@@ -1,11 +1,11 @@
...
 layout: events
 published: true
 
 excerpt: |
-    Joint OSG All Hands Meeting and HTCondor Week July 10-14 2023
+    Save the Dates for **Throughput Computing 2023** - a joint OSG All-Hands Meeting and HTCondor Week to be held July 10-14 2023 in Madison, Wisconsin
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:8]: Why is Dates capitalized? Why is the order of HTCondor Week and OSG All-Hands Meeting reversed here (didn’t Todd want top billing?)?

I think it is traditional to put a comma after the dates: “July 10–14, 2023”, or go with a more international style with “10–14 July 2023”.

File: _events/2023-01-25-htcondor-osg.md [hunk 6]
```diff
@@ -21,13 +21,14 @@ header_image: "/images/capital.jpeg"
 <p style="font-size: larger; font-weight: bold;">Don't miss these in-person learning opportunities in beautiful Madison, Wisconsin!</p>
 
 
-Save the dates! For the first time, the OSG Consortium, the HTCondor team, and the Center for High Throughput Computing will be hosting a Joint HTCondor Week - OSG All-Hands Meeting from July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/).
+Save the dates for **Throughput Computing 2023!** For the first time, we will be offerring **Throughput Computing 2023**, which combines the much loved events **HTCondor Week** and the **OSG All-Hands Meeting** into a new joint event to be held July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/). **Throughput Computing 2023** is sponsored by the OSG Consortium, the HTCondor team, and the UW-Madison Center for High Throughput Computing.
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:24]: Not a fan of the marketing-speak of “the much loved events”.

File: _events/2023-01-25-htcondor-osg.md [hunk 7]
```diff
@@ -21,13 +21,14 @@ header_image: "/images/capital.jpeg"
...
+Save the dates for **Throughput Computing 2023!** For the first time, we will be offerring **Throughput Computing 2023**, which combines the much loved events **HTCondor Week** and the **OSG All-Hands Meeting** into a new joint event to be held July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/). **Throughput Computing 2023** is sponsored by the OSG Consortium, the HTCondor team, and the UW-Madison Center for High Throughput Computing. 
 
-Our hope is that this will primarily be an in-person event, but remote participation (via Zoom) for plenary events will also be offered. Required registration for both components will open in March 2023.
+This will primarily be an in-person event, but remote participation (via Zoom) for the many plenary events will also be offered. Required registration for both components will open in March 2023.
 
  If you register for and travel to the University of Wisconsin in Madison, you can attend plenary and non-plenary sessions, mingle with colleagues, and have planned or ad hoc meetings. In addition, evening events are planned throughout the week.
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:28]: I don’t think these are parallel constructions… if I remove the and clause, I get “If you register for the University …”, which makes no sense. How about “If you register for the in-person event at the University of Wisconsin–Madison, you…”?

**Review by osg-cat — COMMENTED** [review]
There is no mention in this announcement of the co-located ATLAS, CMS, and OSG-supported collaboration meetings. At very least, we should mention the ATLAS and CMS ones. Can you look on previous AHM sites for language? I think it can be just a sentence toward the end of the page.

**Comment by jstathas** [comment]
> There is no mention in this announcement of the co-located ATLAS, CMS, and OSG-supported collaboration meetings. At very least, we should mention the ATLAS and CMS ones. Can you look on previous AHM sites for language? I think it can be just a sentence toward the end of the page.

I added the folloing:The U.S. ATLAS and U.S. CMS high-energy physics projects are also planning parallel OSG-related topics during the event on Wednesday, July 12. (For other attendees, Wedneday's schedule will also include parallel HTCondor and OSG tutorials and OSG Collaborations sessions.)

File: _events/2023-01-25-htcondor-osg.md (hunk 4 above)
Comment (jstathas) [reply _events/2023-01-25-htcondor-osg.md]: I went with the short version

File: _events/2023-01-25-htcondor-osg.md (hunk 5 above)
Comment (jstathas) [reply _events/2023-01-25-htcondor-osg.md]: I added - and also lower cased Dates and added a comma before 2023

File: _events/2023-01-25-htcondor-osg.md (hunk 6 above)
Comment (jstathas) [reply _events/2023-01-25-htcondor-osg.md]: i changed to long-standing

File: _events/2023-01-25-htcondor-osg.md (hunk 4 above)
Comment (jstathas) [reply _events/2023-01-25-htcondor-osg.md]: I used the shorter version

File: _events/2023-01-25-htcondor-osg.md [hunk 8]
```diff
@@ -1,11 +1,11 @@
...
-short_title: Joint OSG All Hands Meeting and HTCondor Week July 10-14
-title: "Save the Dates for the Joint HTCondor Week - OSG All-Hands Meeting - 10-14 July 2023"
+short_title: Announcing Throughput Computing 2023 to be held July 10-14
+title: "Save the dates for Throughput Computing 2023 - a joint HTCondor/OSG event"
 layout: events
 published: true
 
 excerpt: |
-    Joint OSG All Hands Meeting and HTCondor Week July 10-14 2023
+    Save the dates for **Throughput Computing 2023** - a joint OSG HTCondor/OSG event to be held July 10-14, 2023 in Madison, Wisconsin.
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:8]: Am I reading this right? “… a joint OSG HTCondor/OSG event …”? That’s 2 OSGs…

File: _events/2023-01-25-htcondor-osg.md [hunk 9]
```diff
@@ -21,20 +21,23 @@ header_image: "/images/capital.jpeg"
 <p style="font-size: larger; font-weight: bold;">Don't miss these in-person learning opportunities in beautiful Madison, Wisconsin!</p>
 
 
-Save the dates! For the first time, the OSG Consortium, the HTCondor team, and the Center for High Throughput Computing will be hosting a Joint HTCondor Week - OSG All-Hands Meeting from July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/).
+Save the dates for **Throughput Computing 2023!** For the first time, we will be offerring **Throughput Computing 2023**, which combines the long-standing events **HTCondor Week** and the **OSG All-Hands Meeting** into a new joint event to be held July 10–14 at the University of Wisconsin–Madison’s [Fluno Center](https://fluno.com/). **Throughput Computing 2023** is sponsored by the OSG Consortium, the HTCondor team, and the UW-Madison Center for High Throughput Computing.
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:24]: Please find a way to use a spell-checker. “offerring” => “offering”

Also, this sentence is just wordy. I think there is some repetition of combining/joint and events that could be streamlined. How about: “For the first time, **HTCondor Week** and the **OSG All-Hands Meeting** will join together as a single, integrated event from July 10–14 at …”?

File: _events/2023-01-25-htcondor-osg.md [hunk 10]
```diff
@@ -21,20 +21,23 @@ header_image: "/images/capital.jpeg"
...
 
-Topics  will include:
 - Science Enabled by the OSPool and the HTCondor Software Suite (HTCSS)
 - OSG Technology
 - HTCSS  Technology
 - HTCSS and OSG Tutorials
 - State of the OSG
 - Campus  Services and Perspectives
 
+The **U.S. ATLAS and U.S. CMS high-energy physics projects** are also planning parallel OSG-related topics during the event on Wednesday, July 12. (For other attendees, Wedneday's schedule will also include parallel HTCondor and OSG tutorials and OSG Collaborations sessions.)
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:39]: Spell-checker: Wednesday’s

File: _events/2023-01-25-htcondor-osg.md [hunk 11]
```diff
@@ -43,7 +46,9 @@ View last year’s schedules for
 - [HTCondor Week 2022](https://agenda.hep.wisc.edu/event/1733/timetable/#20220523)
 
 {% endcapture %}
+### What
```
Comment (osg-cat) [inline _events/2023-01-25-htcondor-osg.md:49]: From here down seems entirely repetitive.

Comment (osg-cat) [reply _events/2023-01-25-htcondor-osg.md:49]: Except for the Who paragraph. Maybe just move that up and delete the repetitive parts?

File: _events/2023-01-25-htcondor-osg.md (hunk 10 above)
Comment (jstathas) [reply _events/2023-01-25-htcondor-osg.md:39]: fixed

File: _events/2023-01-25-htcondor-osg.md (hunk 11 above)
Comment (jstathas) [reply _events/2023-01-25-htcondor-osg.md:49]: If you are talking about the Where, Who etc. this is repetitive - it is a different presentation of much of the same info intended for those that do not read paragraphs of text. I'm going to leave it - We have been using this format for a while.

**Review by osg-cat — COMMENTED** [review]
I’ve already put a lot of time into this today — I think it would be best to get someone else to do a final review now.

**Comment by CannonLock** [comment]
This was merged in another branch later.

**Pull request closed without merging**